Sources/BraintreeDropIn/Custom Views/BTUIPaymentMethodCollectionViewCell.m

```diff
@@ -82,4 +73,14 @@ -(void)setHighlighted:(BOOL)highlighted {
     self.alpha = highlighted ? 0.5 : 1.0;
 }
 
++ (CGSize)dynamicSize {
```

Comment:
This method creates a fake "sizing cell" so determine the size of a cell based on the current font size. The collection view uses this to determine how tall it should be.

See https://willowtreeapps.com/ideas/dynamic-sizing-for-horizontal-uicollectionviews.